.github/workflows/unstable.yml

```diff
@@ -36,6 +36,20 @@ jobs:
   #
   # Experimental ARC jobs
   #
+  llm-td:
+    name: before-test
+    uses: ./.github/workflows/llm_td_retrieval.yml
+    permissions:
+      id-token: write
+      contents: read
+
+  target-determination:
+    name: before-test
+    uses: ./.github/workflows/target_determination.yml
+    needs: llm-td
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
It would be a good idea to try running ARC versions of these jobs as well.  These target determination jobs have some unusual behaviors and stand a good chance of exposing permissions/access issues